export/securedrop_export/disk/cli.py

```diff
@@ -11,6 +11,9 @@
 
 logger = logging.getLogger(__name__)
 
+# Entries in /dev/mapper on sd-devices
+_DEVMAPPER_SYSTEM = ["control", "dmroot"]
+
```

Comment:
This comment prompts me to ask whether `securedrop-export` as implemented is *necessarily* or only *de facto* coupled to the `sd-devices` VM.  If the former, I'd like to document that coupling in the readme; if the latter, I think we should *avoid* embedding that coupling in code and comments.